source_gen/README.md

```diff
@@ -139,6 +141,11 @@ targets:
         options:
           build_extensions:
             '^lib/{{}}.dart': 'lib/generated/{{}}.g.dart'
+
+      some_cool_builder:
```

Comment:
```suggestion
      # A PartBuilder or LibraryBuilder
      some_cool_builder:
```

I can't comment on the previous config but I would add a comment on that one also that its configuring a `SharedPartBuilder` which uses the combining builder

Reply:
ok, done